src/components/Hoverable/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * It is necessary to create a Hoverable component instead of relying solely on Pressable support for hover state,
+ * because nesting Pressables causes issues. https://github.com/necolas/react-native-web/issues/1875
```

Comment:
NAB, maybe better to summarize exactly what issue they cause rather than force someone to follow this link.